### PR TITLE
Apex CI

### PR DESCRIPTION
**Pull request by milos-ethernal**
Created to test if PR directly from blade repo instead of fork triggers an action with private token because of [this github constraint](https://github.blog/changelog/2021-02-19-github-actions-workflows-triggered-by-dependabot-prs-will-run-with-read-only-permissions/).

If this PR is merged #247 should be closed for this.

### CODE REVIEW

**Review by igorcrevar — APPROVED** [review]

File: .github/workflows/e2e-polybft-test.yml [hunk 1]
```diff
@@ -41,6 +41,20 @@ jobs:
...
+          ref: main
+          token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+      - name: Build apex-bridge
+        run: |
+          go build
+          sudo mv apex-bridge /usr/local/bin/
+      - name: Verify apex-bridge executable
+        run: apex-bridge --help
+      - name: Checkout blade
+        uses: actions/checkout@v4.1.1
```
Comment (bane) [inline .github/workflows/e2e-polybft-test.yml:57]: Reminder: This workflow will never be triggered on the `feature/apex-bridge` branch. Therefore, when we include it in the `ApexBridge CI`, don't forget to add the same `secrets` block as in the `e2e-apex-test.yml` file to ensure it works properly.

**Review by bane — APPROVED** [review]

File: .github/workflows/e2e-apex-test.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
+name: E2E ApexBridge Tests
+on: # yamllint disable-line rule:truthy
+  workflow_call:
+    outputs:
+      workflow_output:
+        description: E2E ApexBridge Tests output
+        value: ${{ jobs.apex.outputs.e2e_apex_output_failure }}
```
Comment (bane) [inline .github/workflows/e2e-apex-test.yml]:
```suggestion
        value: ${{ jobs.e2e_apex_bridge.outputs.e2e_apex_output_failure }} 
```

File: .github/workflows/apex-ci.yml [hunk 3]
```diff
@@ -0,0 +1,91 @@
+name: ApexBridge CI
```
Comment (bane) [inline .github/workflows/apex-ci.yml:2]:
```suggestion
---
name: ApexBridge CI
```

File: .github/workflows/e2e-apex-test.yml [hunk 4]
```diff
@@ -0,0 +1,74 @@
+name: E2E ApexBridge Tests
```
Comment (bane) [inline .github/workflows/e2e-apex-test.yml:2]:
```suggestion
---
name: E2E ApexBridge Tests
```

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -133,6 +133,8 @@ jobs:
       inputs.e2e_polybft_test || 
       (github.event_name == 'pull_request' && github.ref != 'refs/heads/feature/apex-bridge') || 
       (github.event_name == 'push' && (github.ref == 'refs/heads/main' || github.ref == 'refs/heads/develop' || github.ref != 'refs/heads/feature/apex-bridge'))
+    secrets:
+      PERSONAL_ACCESS_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```
Comment (bane) [inline .github/workflows/ci.yml:137]: Reminder: `CI` a.k.a `Blade CI`, will not be triggered on the `feature/apex-bridge` branch. For now, this needs to be in `ApexBridge CI`.

**Review by bane — APPROVED** [review]

**Pull request merged**